src/pages/404.tsx

```diff
@@ -0,0 +1,30 @@
+import Image from "next/image";
+import Link from "next/link";
+
+import { Box, Button, Container, HStack, Text } from "@chakra-ui/react";
+
+function NotFound() {
+  return (
+    <Container
+      display="flex"
+      flexDirection="column"
+      alignItems="center"
+      my="10"
+    >
+      <Box>
+        <Image src="/404.svg" width={500} height={350} alt="404" />
+      </Box>
+      <Text fontSize="3xl" fontWeight="semibold" color="gray.500" my="4">

```

Comment:
Can you refactor this section of the code to look something like the screenshot below?

![image](https://user-images.githubusercontent.com/31373860/187235993-67b58421-3f17-43bb-ab43-dc89ba8f431a.png)

P.S You can use any of the icons from [`react-icons`](https://react-icons.github.io/react-icons/) package.